I wish_If only/maria_ptitsyna.md

```diff
@@ -0,0 +1,10 @@
+1. If only you had come earlier to check out an architecture's lameness suggested of a freshman.
+2. I wish we had waited until his arrival, otherwise we would be in touch with a current project's stage and what we need to do next time. 
+3. I wish we had met with you before leave and discussed your quit reason, I have a lot of thoughts about it and I would like to share them with you.
+4. I regret finishing the project that hadn't been started because of lack of assistants and a consisting 	finance. If it had been fine and dandy, the idea would definitely have been successful on the market and stayed a new blockbuster.
+5. He wishes he hadn't come and led the friend. If I were him, I wouldn't do that. If he had come on time, it wouldn't have happened.
```

Comment:
he wished he hadn't come and brought his friend
I wouldn't have acted like that